aiida/manage/manager.py

```diff
@@ -33,6 +32,16 @@ class Manager:
       * reset manager cache when loading a new profile
     """
 
+    @staticmethod
+    def get_config():
+        """Return the current loaded config, if any
+
+        :return: current loaded config instance
+        :rtype: :class:`~aiida.manage.configuration.config.Config` or None
+        """
```

Comment:
Looking at the docstring of `get_config`, it looks like this function will raise, if the config does not exist (however, it likely gets called in `get_manager` already, to check).
I.e. perhaps the "if any" is not needed here ?

Reply:
The `get_manager` will just return the manager global singleton, creating it if it doesn't exist. Doing so does not require a config nor a loaded profile. i.e. you can do the following in ipython without problem
```
In [2]: from aiida.manage.manager import get_manager                                                                                                                                                             

In [3]: get_manager()                                                                                                                                                                                            
Out[3]: <aiida.manage.manager.Manager at 0x7fd32b9a1f50>

In [6]: get_manager().get_profile() == None                                                                                                                                                                      
Out[6]: True
```
This is why the `get_profile` docstring has the "if any". It is true however, that the `get_config` will raise if it cannot find or parse it, and otherwise will load the config and return it.

I guess updating the docstring of `get_config` to mention the raise, would be sufficient, correct?

Reply:
> I guess updating the docstring of get_config to mention the raise, would be sufficient, correct?

Yep!

Reply:
Done